### ISSUE TITLE
Support LoRA finetuning in the future?

### ISSUE THREAD

**Issue by ZhengJun-AI** [haoheliu/AudioLDM-training-finetuning#15]
Thanks for your great work!
I have made a personal dataset for finetuning, but I think my small dataset will make the model worse. And I find that the LoRA finetuning method is generally applied in diffusion models, which is better than directly finetuning the whole UNet model. Will this repo support LoRA finetuning method in the future?

**Comment by haoheliu** [comment]
Yes that's a great idea. LoRA and other adapter is for sure the good direction to go. I'll implement that when I have time or other guys on GitHub may be interested in contributing to that.
